src/model/resources/storage/long_duration_storage.jl

```diff
@@ -103,7 +103,7 @@ function long_duration_storage!(EP::Model, inputs::Dict)
 					vSOCw[y,r+1] == vSOCw[y,r] + vdSOC[y,dfPeriodMap[r,:Rep_Period_Index]])
 
 	## Last period is linked to first period
-	@constraint(EP, cSoCBalLongDurationStorageEnd[y in STOR_LONG_DURATION, r in MODELED_PERIODS_INDEX[end]],
+	@constraint(EP, cSoCBalLongDurationStorageEnd[y in STOR_LONG_DURATION, r in [MODELED_PERIODS_INDEX[end]]],
 					vSOCw[y,1] == vSOCw[y,r] + vdSOC[y,dfPeriodMap[r,:Rep_Period_Index]])
```

Comment:
Instead of fixing this, I think we should combine the two constraints into one using the `mod1` function:

`r in MODELED_PERIODS_INDEX`
`vSOCw[y, mod1(r+1)]` ...